pkg/app/logger.go

```diff
@@ -8,20 +8,24 @@ import (
 	logging "github.com/op/go-logging"
 )
 
+// LogConfig - The configuration for the logging.
 type LogConfig struct {
 	LogFile string
 	Stdout  bool
 	Level   string
 	Color   bool
 }
 
+// Log - Logging struct that will contain https://godoc.org/os#File and the logging object.
 type Log struct {
 	*logging.Logger
 	file *os.File
 }
 
+// MODULE - Module for the logger.
 const MODULE = "asb"
```

Comment:
Should this be `Module` instead of `MODULE`? What's our stance on const identifier style?